### ISSUE TITLE
Playing Videos causes crash

### ISSUE THREAD

**Issue by Wami007** [danishblunt/SonyAlbumA13Port#6]
Successfully installed apk on 5V but when you try to play video on gallery it's crashing. I think apk needs link video files to internal video player

**Comment by Orof** [comment]
Can confirm, also have this issue with Xperia 1 IV

**Comment by danishblunt** [comment]
> Successfully installed apk on 5V but when you try to play video on gallery it's crashing. I think apk needs link video files to internal video player

No, from the logs it seems some weird state management stuff, easy fix. When I find time I'll fix it.

**Comment by hoppaa** [comment]
Android 15, same crash problem

**Comment by plpawel** [comment]
> > Successfully installed apk on 5V but when you try to play video on gallery it's crashing. I think apk needs link video files to internal video player
> 
> No, from the logs it seems some weird state management stuff, easy fix. When I find time I'll fix it.

I know you stopped developing this app but could you please fix this one thing?
I really like old sony apps, email etc. and they where making the best live wallpapers (cosmic flow) and some of them stopped installing on A15 :(
